src/storage/src/hummock/sstable/builder.rs

```diff
@@ -102,14 +107,16 @@ impl SSTableBuilder {
 
         // TODO: refine me
         let mut raw_value = BytesMut::default();
-        value.encode(&mut raw_value);
+        let meta = value.encode(&mut raw_value);
```

Comment:
Suggest rename to `value_meta` to avoid ambiguity.

Reply:
fixed